Source/WebCore/rendering/RenderReplaced.cpp

```diff
@@ -699,9 +702,10 @@ void RenderReplaced::computePreferredLogicalWidths()
 {
     ASSERT(preferredLogicalWidthsDirty());
 
-    // We cannot resolve any percent logical width here as the available logical
-    // width may not be set on our containing block.
-    if (style().logicalWidth().isPercentOrCalculated())
+    // We cannot resolve some logical width here (i.e. percent, fill-available or fit-content)
+    // as the available logical width may not be set on our containing block.
+    const auto& logicalWidth = style().logicalWidth();
+    if (logicalWidth.isPercentOrCalculated() || logicalWidth.isFillAvailable() || logicalWidth.isFitContent())
```

Comment:
This looks like a mistake, right? Surely this is intended for another pull request?

Reply:
Sorry about that! I have no idea how this got in. Must have been a mistake while I was rebasing. I'll scroll through it one more time and make sure there's nothing else like this.